### ISSUE TITLE
hash/maphash: malformed module path "hash/maphash": missing dot in first path element

### ISSUE THREAD

**Issue by nsingxia** [mattermost/mattermost-server#15337]
Per Mattermost guidelines, GitHub issues are for bug reports: <http://www.mattermost.org/filing-issues/>.

For troubleshooting see: http://forum.mattermost.org/.
For feature proposals see: http://www.mattermost.org/feature-requests/

If you've found a bug--something appears unintentional--please follow these steps:

1. Confirm you’re filing a new issue. [Search existing tickets in Jira](https://mattermost.atlassian.net/issues/) to ensure that the ticket does not already exist.
2. Confirm your issue does not involve security. Otherwise, please see our [Responsible Disclosure Policy](https://about.mattermost.com/report-security-issue/).
3. [File a new issue](https://github.com/mattermost/mattermost-server/issues/new) using the format below. Mattermost will confirm steps to reproduce and file in Jira, or ask for more details if there is trouble reproducing it. If there's already an existing bug in Jira, it will be linked back to the GitHub issue so you can track when it gets fixed. 

#### Summary
Bug report in one concise sentence

#### Steps to reproduce
How can we reproduce the issue (what version are you using?)

#### Expected behavior
Describe your issue in detail

#### Observed behavior (that appears unintentional) 
What did you see happen? Please include relevant error messages and/or screenshots.

#### Possible fixes
If you can, link to the line of code that might be responsible for the problem

**Comment by jasonblais** [comment]
@nsingxia Are you open to sharing the information above (steps to reproduce, expected and observed behaviour)? This will make it easier for us to troubleshoot the issue.

**Comment by sirMackk** [comment]
I bumped into the same issue after cloning https://github.com/mattermost/mmctl and running `make test`. It also happened when running `make build`, `go mod tidy` and `go mod vendor`.

A search brought me to https://community.k6.io/t/building-from-source-seems-to-thrown-an-exception/876/2, which suggested upgrading my golang install. It made sense because the test/build commands succeed for other `mmctl` developers, which indicated the error had something to do with my go set up. I was using 1.13.8 and had been thinking of upgrading for a some time.

I upgraded my go install to `1.15.2` and everything works fine now. Hope this helps anyone else that bumps into this error.
